packages/expo-updates/android/src/main/java/expo/modules/updates/reloadscreen/ReloadScreenConfiguration.kt

```diff
@@ -0,0 +1,82 @@
+package expo.modules.updates.reloadscreen
+
+import android.net.Uri
+import expo.modules.kotlin.records.Field
+import expo.modules.kotlin.records.Record
+import expo.modules.kotlin.types.Enumerable
+import androidx.core.graphics.toColorInt
+
+data class ReloadScreenOptions(
+  @Field val backgroundColor: String?,
+  @Field val image: ReloadScreenImageSource?,
+  @Field val imageResizeMode: ImageResizeMode?,
+  @Field val imageFullScreen: Boolean?,
+  @Field val fade: Boolean?,
+  @Field val spinner: SpinnerOptions?
+) : Record
+
+data class ReloadScreenImageSource(
+  @Field val url: Uri? = null,
```

Comment:
I wonder if this should be required to be a local asset to ensure it doesn't slow down the reload?